sys/include/xtimer.h

```diff
@@ -280,6 +281,13 @@ int xtimer_msg_receive_timeout(msg_t *msg, uint32_t us);
 int xtimer_msg_receive_timeout64(msg_t *msg, uint64_t us);
 
 /**
+ * @brief Get the current time in seconds and microseconds since system start
+ *
+ * @param[in] tp    Uptime will be stored in the timeval structure pointed to by tp
+ */
+void xtimer_gettimeofday(struct timeval *tp);
```

Comment:
@kaspar030 sorry, I reverted the commit that introduced the API CHANGE and didn't saw your comments anymore.
> got it, I'll PR newlib support for gettimeofday...

nice.